packages/engine/src/avatar/systems/AvatarTransparencySystem.tsx

```diff
@@ -100,6 +101,8 @@ export const AvatarTransparencySystem = defineSystem({
 
 const AvatarReactor = (props: { entity: Entity }) => {
   const entity = props.entity
+  if (!hasComponent(entity, UUIDComponent)) return null // Early exit if the UUID on this avatar has been removed, which can happen when another player leaves the scene
```

Comment:
a better fix would be to just have a `useComponent(entity, UUIDComponent)` which will suspend the context if the component doesn't exist